styles/style.css

```diff
@@ -78,6 +78,28 @@ body {
 	text-transform: uppercase;
 }
 
+.my-container {
+	width: 100%;
+	height:auto;
```

Comment:
You don't need this because you're already using Bootstrap's `container` class. All you need is to add the `padding-bottom`, and you should do that in a more descriptive class name.